### PR TITLE
Event: Make event aliases optional in unit tests

### PR DESCRIPTION
**Pull request by dmethvin**
Relatively easy fixes, although I did have a question about one strange VML-related test.

### CODE REVIEW

File: test/unit/event.js [hunk 1]
```diff
@@ -900,49 +900,6 @@ QUnit.test( "mouseenter, mouseleave don't catch exceptions", function( assert )
 	}
 } );
 
-if ( jQuery.fn.click ) {
```
Comment (dmethvin) [inline test/unit/event.js]: Just moved this down and made it a `QUnit.skip` test.

**Comment by timmywil** [comment]
LGTM

File: test/unit/event.js [hunk 2]
```diff
@@ -2843,6 +2800,46 @@ QUnit.test( "originalEvent property for Chrome, Safari, Fx & Edge of simulated e
...
+	assert.equal( counter, 1, "Check that click, triggers onclick event handler also" );
+
+	clickCounter = 0;
+	jQuery( "#simon1" )[ 0 ].onclick = function() {
+		clickCounter++;
+	};
+	jQuery( "#simon1" ).click();
+	assert.equal( clickCounter, 1, "Check that click, triggers onclick event handler on an a tag also" );
+
+	// test that special handlers do not blow up with VML elements (#7071)
```
Comment (dmethvin) [inline test/unit/event.js]: This doesn't look like it belongs on an event alias unit test. It also is appending to head and not removing the markup. Do we need this test at all now? @timmywil do you remember anything about https://github.com/jquery/jquery/commit/69866fd2e5ba2456c2ae84a5ba711a1db2a25091 ?

Comment (timmywil) [inline test/unit/event.js]: I don't recall the specifics, but it looks like we had that xml in the index.html to start. That said, I think it's safe to remove at the end of the test. Regardless, I agree that it should be moved to non-alias tests.

**Comment by mzgol** [comment]
LGTM.

**Pull request merged**